freqtrade/optimize/backtesting.py

```diff
@@ -1070,6 +1077,26 @@ def backtest_one_strategy(self, strat: IStrategy, data: Dict[str, DataFrame],
         })
         self.all_results[self.strategy.get_strategy_name()] = results
 
+        if self.backtest_signal_candle_export_enable and \
```

Comment:
This should be moved to a separate function that's only called here.

It's not "really" needed by `backtest_one`, so it should be treated separately i think.
it'll also make the dependencies clearer (what's needed as input, ...) - as these have to be passed in as arguments.

Reply:
ok I'll move it out

Reply:
Implemented in [f92997d](https://github.com/freqtrade/freqtrade/pull/6543/commits/f92997d3789c6eb8b9116e4a55dac2b9b7112aa7)

Reply:
Oh, or did you mean move it out completely and generate it outside `backtest_one`?

Reply:
i think you will have to call it from here ... so my intend is what you implemented... stop doubing yourself after implementing correctly ;)